tests/foreman/ui/test_ansible.py

```diff
@@ -182,7 +183,7 @@ def _finalize():
         result = rhel_contenthost.execute(command)
         assert result.status == 0, f'Failed to register host: {result.stderr}'
         target_host = rhel_contenthost.nailgun_host
-        default_value = '[\"test\"]'
+        default_value = '["test"]'
```

Comment:
I'd ask @shweta83 to confirm if this could change test behavior; she originally wrote this test and tried this string.

Reply:
For Python interpreter, both are the same. Therefore this change should not have any negative effects. 

```python
[ins] In [1]: '[\"test\"]' == '["test"]'
Out[1]: True
```

Reply:
I tried with interpreter and it worked but somehow when it fails to read as a string when read from the page. So I added this.

Reply:
I have reverted the patch in 9d648a751733570d4fe4b5dd3925cd240342caac and I have added a comment that will prevent formatted from touching this chunk of code.

Reply:
Thank you @ogajduse